recipes/responder/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "responder" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c2232012ce346104a4f0e225731cab24bf80ba460b8636d1a6c05343d1c95b52
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - starlette
+    - uvicorn
+    - aiofiles
+    - pyyaml
+    - requests
+    - graphene
+    - graphql-server-core >=1.1
+    - jinja2
+    - parse
+    - uvloop  # [not win]
+    - rfc3986
+    - python-multipart
+    - chardet
+    - apispec >=1.0.0b1
+    - marshmallow >=3.0.0b7
+    - asgiref
+    - docopt
+    - itsdangerous
+    - requests-toolbelt
+    - apistar
+
+test:
+  imports:
+    - responder
+
+about:
+  home: https://github.com/kennethreitz/responder/
+  license: APACHE-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: 'A familiar HTTP Service Framework for Python'
+  doc_url: http://simplejson.readthedocs.io/
```

Comment:
left-over from the example?

Reply:
thanks @ltalirz! @dhirschfeld can you please fix?

Reply:
Could be a good opportunity to try the [suggested changes](https://blog.github.com/changelog/2018-10-16-suggested-changes/)